cli/spark/endpoints/init.py

```diff
@@ -8,14 +8,29 @@
 def setup_parser(parser: argparse.ArgumentParser):
     parser.add_argument('--global', dest='global_flag', action='store_true',
                         help="Create a .aztk/ folder in your home directory for global configurations.")
+    software_parser = parser.add_mutually_exclusive_group()
+    software_parser.add_argument('--python', action="store_true", required=False)
+    software_parser.add_argument('--r-base', action="store_true", required=False)
```

Comment:
Maybe just call it 'r' instead of 'r-base'? My guess is that these should be customer/solution focused, not based on our docker images.